src/Serialization/Upgrades/main.jl

```diff
@@ -172,6 +177,15 @@ function upgrade(format_version::VersionNumber, dict::Dict)
       # upgrading large files needs a work around since the new load
       # uses JSON3 which is read only 
       upgraded_dict = upgrade_script(upgrade_state, upgraded_dict)
+      if script_version > v"0.13.0"
+        if haskey(upgraded_dict, :_refs)
+          upgraded_refs = Dict()
```

Comment:
Can't we make the type of this `Dict` more concrete? I am caring less about the performance here (although that of course also matters) and more about the readability & understandability of the code. As it is I have no idea what the key types of this are.